pkg/services/ngalert/api/tooling/api.json

```diff
@@ -505,6 +505,12 @@
    "type": "object",
    "x-go-package": "github.com/prometheus/common/config"
   },
+  "ConfFloat64": {
```

Comment:
Sorry that this whole file is such a mess. Some of it comes from adding the DataResponse type and its dataframes to the API spec, and some of it is the usual "randomness" of swagger generation we see.